### PR TITLE
Pause feature

### PR DESCRIPTION
**Pull request by GeorgeNance**
Added a prop to easily pause the animation.

### CODE REVIEW

File: src/index.vue [hunk 1]
```diff
@@ -14,6 +14,14 @@ export default {
     delay: {
       type: String,
       default: '0s'
+    },
+        delay: {
```
Comment (biigpongsatorn) [inline src/index.vue]: `delay` is duplicate

File: src/index.vue [hunk 2]
```diff
@@ -22,7 +30,8 @@ export default {
         ...this.barStyle,
         'animation-duration': this.duration,
         'animation-direction': this.direction,
-        'animation-delay': this.delay
+        'animation-delay': this.delay,
+        'animation-play-state' : (this.paused) ? "paused" : "running"
```
Comment (biigpongsatorn) [inline src/index.vue]: please use single-quote `'` instead of a double-quote `"`.

File: src/index.vue (hunk 1 above)
Comment (GeorgeNance) [reply src/index.vue]: Removed duplicate delay

File: src/index.vue (hunk 2 above)
Comment (GeorgeNance) [reply src/index.vue]: Done

**Pull request merged**